libraries/freertos_plus/standard/crypto/src/iot_crypto.c

```diff
@@ -256,7 +256,7 @@ void CRYPTO_ConfigureHeap( void )
     /*
      * Ensure that the FreeRTOS heap is used.
      */
-    mbedtls_platform_set_calloc_free( prvCalloc, vPortFree ); /*lint !e534 This function always return 0. */
+    //mbedtls_platform_set_calloc_free( prvCalloc, vPortFree ); /*lint !e534 This function always return 0. */
```

Comment:
This change won't be necessary once this PR has been rebased to the master branch